dora/core/server/worker/src/main/java/alluxio/worker/http/HttpLoadService.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.worker.http;
+
+import alluxio.AlluxioURI;
+import alluxio.client.file.FileSystem;
+import alluxio.grpc.JobProgressReportFormat;
+import alluxio.grpc.LoadJobPOptions;
+import alluxio.job.JobDescription;
+import alluxio.job.LoadJobRequest;
+
+import io.grpc.Status;
+import io.grpc.StatusRuntimeException;
+
+import java.util.Optional;
+
+/**
+ * This service allows to submit, stop, and get the progress of the load job by HTTP RESTful API.
+ */
+public class HttpLoadService {
+
+  private static final JobProgressReportFormat DEFAULT_FORMAT = JobProgressReportFormat.TEXT;
+
+  private static final String JOB_TYPE = "load";
```

Comment:
stringly typed? is there an enum for this?

Reply:
Updated. I used enum instead.